lib/ansible/module_utils/csharp/Ansible.Basic.cs

```diff
@@ -800,6 +818,11 @@ private void SetArgumentSpecDefaults(IDictionary argumentSpec)
                         string msg = "A deprecated_aliases date must be a DateTime object";
                         throw new ArgumentException(FormatOptionsContext(msg, " - "));
                     }
+                    string collectionName = null;
+                    if (depInfo.ContainsKey("collection_name"))
```

Comment:
Would you be able to add `collection_name` to the [deprecated_aliases doc](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/developing_modules_general_windows.rst#windows-new-module-development).

Reply:
I added it, and also `removed_from_collection`.